spaceship/lib/spaceship/test_flight/build.rb

```diff
@@ -119,6 +119,11 @@ def self.all_processing_builds(app_id: nil, platform: nil, retry_count: 0)
         all(app_id: app_id, platform: platform, retry_count: retry_count).find_all(&:processing?)
       end
 
+      # Just the builds, as a flat array, that are waiting for beta review
+      def self.all_waiting_for_review(app_id: nil, platform: nil, retry_count: 0)
```

Comment:
Is it possible for multiple builds to be waiting for review?

Reply:
https://github.com/fastlane/fastlane/pull/12565#discussion_r192190395

Reply:
Nice! That is my understanding as well. I was just thinking that `all` in `all_waiting_for_review` can throw someone off. may be indicate that there can only be one by the method name too?